presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/TransformQuantifiedComparisonApplyToScalarApply.java

```diff
@@ -191,10 +221,32 @@ private PlanNode rewriteQuantifiedOrderableApplyNode(ApplyNode node, QuantifiedC
                     node.getId(),
                     context.rewrite(node.getInput()),
                     subqueryPlan,
-                    Assignments.of(subValue, subValue.toSymbolReference()),
+                    Assignments.of(extremeValue, extremeValue.toSymbolReference()),
```

Comment:
This is changed later on to 
`Assignments.of(minValue, minValue.toSymbolReference(), maxValue, maxValue.toSymbolReference())`.
Added an `Assingments.identity(Symbol...)` overload and changed to `Assignments.identity(minValue, maxValue)`.